crates/nu-command/Cargo.toml

```diff
@@ -96,6 +96,7 @@ url = "2.2"
 uuid = { version = "1.3", features = ["v4"] }
 wax = { version = "0.5" }
 which = { version = "4.4", optional = true }
+nu-cmd-base = { version = "0.80.1", path = "../nu-cmd-base" }
```

Comment:
Watch out: this is behind the main versions and would get missed by our regex version bump

Reply:
very good catch :+1: 
i think i did create this crate before the last release :open_mouth:

Reply:
it should be a lot better in 4de6bf7cd8c3565450ef23e689e1e5caf2a69214 :wink: